### DISCUSSION TITLE
Allow exceptions when limiting branches/tags updated in a single push

### DISCUSSION THREAD

**Discussion by skeggse** [community/community#40660] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

We've been using the repository option for limiting the number of branches/tags that can be updated in a single push. Not sure when it got enabled, but it silently broke the script we run to delete old tags, meaning we've accumulated a lot of excess tags. Is there a way to except specific users/bots from the restriction, and if not, could we consider adding an exception to the roadmap?

![image](https://user-images.githubusercontent.com/1348991/205405386-907c6855-bc4f-446f-b815-c036e5041934.png)

**Comment by ossanna16** [comment]
Hi there @skeggse and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
